js/bitstamp.js

```diff
@@ -772,11 +944,28 @@ module.exports = class bitstamp extends Exchange {
         return this.parseTransactions (transactions, currency, since, limit);
     }
 
+    async fetchDeposits (code = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privatePostUserTransactions (this.extend (request, params));
+        let currency = undefined;
+        if (code !== undefined) {
+            currency = this.currency (code);
+        }
+        const transactions = this.filterBy (response, 'type', '0');
+        return this.parseTransactions (transactions, currency, since, limit);
```

Comment:
Do you mind if we exclude `fetchDeposits` from this PR? Since the exchange does not have an endpoint for fetching the deposits-only, it will mix deposits with other transactions in its responses, and that will effectively break the pagination ( https://github.com/ccxt/ccxt/wiki/Manual#pagination)

Apart from the above, I think this PR is complete.

Reply:
It's fine, thank you!

Reply:
@domenus thx to you for your involvement, appreciate your help!